docs/components/InteractiveWell.tsx

```diff
@@ -0,0 +1,135 @@
+import classNames from "classnames";
+import { useGlowContext, GlowSignInButton } from "@glow-app/glow-react";
+import { NetworkSwitcher } from "./atoms/NetworkSwitcher";
+import { LuxButton } from "./LuxButton";
+
+export const InteractiveWell = ({
+  title,
+  minimal = false,
+  className = "",
+  children,
+}: {
+  title: string;
+  // If true, this hides the network switcher and disconnect button.
+  // Useful for cleaner success states.
+  minimal?: boolean;
+  className?: string;
+  children: React.ReactNode;
+}) => {
+  const { user, glowDetected, signOut } = useGlowContext();
+
+  const blurred = !glowDetected || !user;
+
+  return (
+    <section className={classNames("px-3 pb-3 rounded", className)}>
+      <div className="title text-xs font-weight-bold">{title}</div>
+
+      {!minimal && (
+        <div className={classNames("network-switcher", { blurred })}>
+          <NetworkSwitcher />
+        </div>
+      )}
+
+      <div className={classNames("children", { blurred })}>{children}</div>
+
+      {!minimal && (
+        <div className={classNames("disconnect", { blurred })}>
```

Comment:
I'm not a fan of absolute positioning the `Disconnect` buttons since that feels pretty brittle and the Disconnect button isn't part of the chrome of the component, it's part of the inside of the component. Let's move it back to the parent